src/imputation/imputation_main.py

```diff
@@ -53,13 +58,28 @@ def run_imputation(
     for col in orig_cols:
         df[f"{col}_imputed"] = df[col]
 
+    # Create imp_path variable for QA output and manual imputation file
+    NETWORK_OR_HDFS = config["global"]["network_or_hdfs"]
+    imp_path = config[f"{NETWORK_OR_HDFS}_paths"]["imputation_path"]
+
+    # Load manual imputation file
+    df, manually_trimmed_df = mimp.merge_manual_imputation(
+        df, config, is_file, read_csv, imp_path
```

Comment:
I can see you're in the middle of updating this: you don't need to read in the csv any more as already done